sensor_test.py

```diff
@@ -129,40 +137,40 @@ def test_update_sensor_firmware(get_sensor_info, update_sensor_firmware):
     """
     max_firmware_version = 15
 
-    print('Step 1: Get original sensor firmware version')
+    log.info(f"Get original sensor firmware version")
     current_sensor_fw_ver = get_sensor_info().get("firmware_version")
 
     while current_sensor_fw_ver < max_firmware_version - 1:
         sensor_fw_ver_before_update = current_sensor_fw_ver
 
-        print('Step 2: Request firmware update')
+        log.info(f"Request firmware update")
         sensor_fw_update_request = update_sensor_firmware()
```

Comment:
Our factory fixture `update_sensor_firmware()` now returns a function that performs an end-to-end firmware update. It does the following:
1. Request update from sensor.
2. Validate the response from sensor is correct.
3. Wait for sensor to come back online after the update.

So now steps 2 and 3 in the function above overlap with what the test is doing. This is what I meant when I said that we'll need to modify the test in order to account for changes in factory fixtures. `update_sensor_firmware()` now does all the work for us, so we can remove duplicate parts (validating initial response and waiting) from the test.

Reply:
i'm resolving issue according to LMS